### PR TITLE
#167484574 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by harerakalex**
### What does this PR do?
- Setup TravisCI and add badge in readme
### Description of Task to be done?
- Integrate TravisCi with Github.
- Add markdown from TravisCI into readme to create a badge.
### How should this be manually tested?
- Right now, Go to this branch `ch-setup-travis-ci-167484574` you will see the  badge like this [Build Status](https://travis-ci.org/andela/coding-geeks-ah-backend) on readme
### Any background context you want to provide?
- N/A
### What are the relevant pivotal tracker stories?
- [Integrate TravisCI with readme badge](https://www.pivotaltracker.com/story/show/167484574)
### Questions
N/A

### CODE REVIEW

**Comment by malaba6** [comment]
Carlos does your PR need reviews or not? Because I don't see any labels here

**Comment by harerakalex** [comment]
> Carlos does your PR need reviews or not? Because I don't see any labels here

Thank you, Yes It needs reviews please.

**Comment by Malaba6** [comment]
The PT story's ID at the beginning of the PR title doesn't look right to me. If you could move it at 
the end!

**Review by malaba6 — COMMENTED** [review]
Is there a way to make your tests pass? because they are currently failing

**Comment by harerakalex** [comment]
> Is there a way to make your tests pass? because they are currently failing

Currently the Travis CI build is failing because unit test environment is not yet set up.

**Comment by raymond42** [comment]
> Is there a way to make your tests pass? because they are currently failing

**Comment by raymond42** [comment]
> Is there a way to make your tests pass? because they are currently failing

I think right now they can't pass because we have not implemented our tests yet even if we implement the tests they should be failing till we implement working functions to pass those tests for example if we say that this function should be this and this for the first time because we do not have those functions yet they should not be passing but If we implement those functions it will pass, but for now I think they should be failing

**Review by raymond42 — COMMENTED** [review]
All good for me

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+node_js:
+  - 'stable'
+install:
+  - npm install
+services:
+  - postgresql
+
+before_script:
+  - psql -c 'create database authors_haven_test;' -U postgres
+
```
Comment (mkiterian) [inline .travis.yml:12]: add a command to build the app.

**Review by mkiterian — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (harerakalex) [reply .travis.yml:12]: I just put it here
Screen Shot 2019-07-31 at 14 11 52

**Review by Jkadhuwa — APPROVED** [review]
Everything looks okay to me

**Review by mkiterian — APPROVED** [review]
Looks good

**Pull request merged**